visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigItemGroup/customConfigDescription/customConfigDescription.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="config-description-container">
+	<p><b>Name: </b>{{ customConfigItem.name }}</p>
+	<p><b>Selection Mode:</b> {{ customConfigItem.mapSelectionMode | titlecase }}</p>
+	<p><i class="fa fa-arrows-alt"></i> {{ customConfigItem.metrics.areaMetric }}</p>
+	<p><i class="fa fa-arrows-v"></i> {{ customConfigItem.metrics.heightMetric }}</p>
```

Comment:
What about the selected color and edge metric?
I would suggest to just display them even if they are the same.